.circleci/ci-packages.txt

```diff
@@ -110,11 +110,8 @@ perl-HTTP-Daemon-6.01
 perl-HTTP-Date-6.02
 perl-HTTP-Message-6.14
 perl-HTTP-Negotiate-6.01
-perl-HTTP-Parser-XS-0.17
 perl-Import-Into-1.002005
 perl-indirect-0.38
-perl-Inline-0.80
-perl-Inline-Python-0.56
```

Comment:
I don't think we should have lost most of these packages and a missing `perl-Inline-Python` would explain why the fullstack test fails here.